bootstrap/hook.go

```diff
@@ -105,6 +109,15 @@ func newHookScriptWrapper(hookPath string) (*hookScriptWrapper, error) {
 			"SET " + hookWorkingDirEnv + "=%CD%\n" +
 			"SET > \"" + h.afterEnvFile.Name() + "\"\n" +
 			"EXIT %" + hookExitStatusEnv + "%"
+	} else if runtime.GOOS == "windows" && isPwshHook {
+		script = "@echo off\n" +
+			"SETLOCAL ENABLEDELAYEDEXPANSION\n" +
+			"SET > \"" + h.beforeEnvFile.Name() + "\"\n" +
+			"powershell.exe -ExecutionPolicy Bypass -NoProfile -NonInteractive -File \"" + absolutePathToHook + "\"\n" +
+			"SET " + hookExitStatusEnv + "=!ERRORLEVEL!\n" +
+			"SET " + hookWorkingDirEnv + "=%CD%\n" +
+			"SET > \"" + h.afterEnvFile.Name() + "\"\n" +
+			"EXIT %" + hookExitStatusEnv + "%"
```

Comment:
The purpose of these wrappers is to execute the provided command script in the same context and capture any different to the environment files so we can fold those changes into the parent env and consecutive hooks and scripts — is there a way to do this in powershell, too?